flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_BrakeTemp.js

```diff
@@ -82,9 +82,9 @@ class A32NX_BrakeTemp {
                 SimVar.GetSimVarValue("L:A32NX_REPORTED_BRAKE_TEMPERATURE_4", "celsius")
             ];
         }
-        const GearLeftPosition = SimVar.GetSimVarValue("GEAR LEFT POSITION", "Percent Over 100");
+        const GearLeftPosition = SimVar.GetSimVarValue("L:A32NX_GEAR_LEFT_POSITION", "Percent Over 100");
```

Comment:
This requires line 91 to be changed to:
`        const brakeFanIsOn = brakeFanButtonIsPressed && (GearLeftPosition === 100);
`

otherwise the BRK FAN breaks

Reply:
Thanks, updated!